src/docfx/docfx.csproj

```diff
@@ -18,26 +18,31 @@
     <!-- To keep HTML intact during v3 migration, keep using HtmlAgilityPack 1.4.9 -->
     <PackageReference Include="HtmlAgilityPack" Version="1.4.9" />
     <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.298" />
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.11.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.12.0" />
     <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```

Comment:
This is still using ASPNET Core 2, interestingly APSNET Core 3 doesn't ship as NuGet package: https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#remove-obsolete-package-references